src/components/ButtonGroup.scss

```diff
@@ -0,0 +1,35 @@
+// * ButtonGroup
+
+$ButtonGroup-border-radius: $border-radius-medium;
+$ButtonGroup-space: -1 * $border-width-small;
+
+
+// * 1. Lay out items left to right
+// * 2. Make IE10 play nice
+
+.ButtonGroup {
+  display: flex; // 1
+  display: -ms-flexbox; // 2
+}
+
+
+// * 1. Overlap elements and override any vertical margins.
+// * 2. Reset `border-radius` so corners line up.
+
+.ButtonGroup > * {
```

Comment:
Couldn't this just be .ButtonGroup > .Button, I'm guessing the button class will always be the direct children of button right?